.expeditor/verify.pipeline.yml

```diff
@@ -33,6 +33,14 @@ steps:
       docker:
         image: ruby:2.6-stretch
 
+- label:  run-tests-ruby-2.7-preview
+  command:
+    - /workdir/.expeditor/buildkite/verify.sh
```

Comment:
Probably should be `/workdir/.expeditor/buildkite/verify.sh || true` for now.

Reply:
Fixed =)